app/services/ccms/attribute_value_generator.rb

```diff
@@ -86,6 +86,10 @@ def used_delegated_functions_on(_options)
       @legal_aid_application.used_delegated_functions_on.strftime('%d-%m-%Y')
     end
 
+    def app_ammendment_type(_options)
```

Comment:
Spelling: `app_ammendment_type` should be `app_amendment_type`. You'll have to make changes to `config/ccms/ccms_keys.yml` as a result.

Reply:
thanks, at least its not like ccms where we have to live with the spelling errors